Client/mods/deathmatch/logic/luadefs/CLuaEngineDefs.cpp

```diff
@@ -14,6 +14,13 @@
 #include <game/CObjectGroupPhysicalProperties.h>
 #include <game/CStreaming.h>
 #include <lua/CLuaFunctionParser.h>
+#include "../../../core/CModelCacheManager.h"
```

Comment:
That's just lazy. Please use an interface class for this (place it in `Client/sdk/core/` and inherit from it).

Reply:
That's how it was done before me too.